packages/commonwealth/package.json

```diff
@@ -187,6 +189,7 @@
     "jdenticon": "^2.1.1",
     "jsdom-global": "^3.0.2",
     "jsonwebtoken": "^9.0.0",
+    "lexical": "^0.17.0",
```

Comment:
is it necessary to pull in lexical?

Reply:
Unfortunately, yes.  There are 2-3 imports we have to use.